packages/imhex.vm/imhex.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>imhex.vm</id>
-    <version>1.34.0</version>
+    <version>1.35.4</version>
```

Comment:
This version doesn't run properly in a VBox Windows 10 VM (OS Build: `19045.2006`)

![error_imhex](https://github.com/user-attachments/assets/27f1baa2-22d3-4863-b45d-221c1d39f554)

Reply:
Fixed at d5f5c333de7233e9e53f2be62d5e61f08a7ca6fc.

Reply:
Is there a PR for this fix?